server/src/database.js

```diff
@@ -35,11 +64,14 @@ const createStore = () => {
         email: {
           type: Sequelize.STRING, allowNull: false,
         },
-        token: Sequelize.STRING,
         birthday: Sequelize.DATE,
         city: Sequelize.STRING,
         state: Sequelize.STRING,
+        zip: Sequelize.STRING,
+        additionalAddress: Sequelize.STRING,
         phone: Sequelize.STRING,
+        description: Sequelize.STRING,
```

Comment:
What goes into description for User? Description is a poorly defined concept and all kinds of data can creep into this. Let's either rename to some more specific purpose, or remove this field.